en/hubs_tos.md

```diff
@@ -0,0 +1,82 @@
+# Terms of Service for Hubs
+
+Version 1.0, Effective October 11, 2022
+{: datetime="2022-10-11" }
+
+Hubs (the “Service”) is a new, preview service that allows you to set up and manage your own set of real-time communications platforms for virtual reality, augmented reality, desktop, laptop, or mobile. It is available as a paid subscription service.  
+
+These Terms of Service explain your rights and responsibilities when you use Hubs. By purchasing and using Hubs, you assure Mozilla that you have read these terms and agree to them. 
+
+## 1. Subscribing to Hubs
+
+### You’ll Need A Firefox Account
+A Firefox Account is required to use the Service. To create a Firefox Account, you will also need to agree to the [Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Privacy Notice](https://www.mozilla.org/privacy/firefox/) for your Firefox Account.
```

Comment:
lower case `account` as in `Firefox account`

Reply:
fixed